.gitignore

```diff
@@ -0,0 +1,11 @@
+/.bundle/
+/doc/
+/log/*.log
+/pkg/
+/tmp/
+/test/dummy/db/*.sqlite3
+/test/dummy/db/*.sqlite3-*
+/test/dummy/log/*.log
+/test/dummy/storage/
+/test/dummy/tmp/
```

Comment:
@MohammedNazeer10  What is the purpose of having dummy folder?

Reply:
@rajeshkanna-mallow , it is created by default in rails engine for the purpose of testing and verifying the functionality.